project_one.html

```diff
@@ -11,23 +11,46 @@
       href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     <title>Saifullah portfolio</title>
   </head>
   <body>
     <section class="container">
       <div class="pageone">
-        <header class="header">
+        <header class="header" id="pageone">
           <nav class="logo">
             <a href="#" class="logo1"> My Logo </a>
-            <img class="menu" src="./icons/Menu.png" alt="Menu button" />
+            <a
+              class="menu icon"
+              id="menu"
+              href="javascript:void(0);"
+              onclick="menu_btn()"
+            >
+              <img src="./icons/Menu.png" alt="Menu button" />
+            </a>
             <ul>
-              <li><a href="#" class="focusable">Portfolio</a></li>
-              <li><a href="#" class="focusable">About</a></li>
-              <li><a href="#" class="focusable">Contact</a></li>
+              <li><a href="#portfolio" class="focusable">Portfolio</a></li>
+              <li><a href="#about" class="focusable">About</a></li>
+              <li><a href="#contact" class="focusable">Contact</a></li>
             </ul>
           </nav>
         </header>
-        <section class="content">
+        <div id="myLinks">
+          <a href="javascript:void(0);" class="icon" onclick="menu_btn()">
+            <img src="./images/Close_menu.svg" alt="close botton" />
+          </a>
```

Comment:
- [ ] Kindly this icon has to have a `display:none` on the desktop version. To just appear on the mobo version.

![image](https://user-images.githubusercontent.com/35535130/161645480-9d4f22ba-c026-49dc-b63e-f3a09aad0812.png)